app/models/concerns/text_sanitizer.rb

```diff
@@ -37,4 +42,20 @@ def sanitize_category(kw, max_length, strict)
       kw.strip.slice(0, max_length)
     end
   end
+
+  def add_newlines_to_tags(text)
+    text.gsub(/<p>/i, "\n<p>")
+      .gsub(/<\/p>/i, "</p>\n")
+      .gsub(/<div>/i, "\n<div>")
+      .gsub(/<\/div>/i, "</div>\n")
+      .gsub(/<br>/i, "\n<br>")
+      .gsub(/<br\s*\/>/i, "\n<br>")
+  end
+
+  def clean_whitespace(text)
+    text.tr("\r", "\n")
+      .gsub(/\s*\n\s*/, "\n")
```

Comment:
the goal here is to end up with a single newline, even if there were multiple p or br tags stacked
arguably, we could support multiple newlines...that would be another way to go, but I don't see a good reason for it in the text versions of these descriptions